app/Locale/uk_UA/translations.php

```diff
@@ -1241,7 +1241,7 @@
     'This username is already taken' => 'Цей логін вже зайнято',
     'A link to reset your password has been sent by email.' => 'Посилання для скидання пароля було надіслано на ваш E-mail.',
     'Your profile must have a valid email address.' => 'Ваш профіль повинен мати правильний E-mail.',
-    'Unfortunately, we are unable to reset your password. Did you enter a valid username? Do you have an email address in your profile?' => 'На жаль, не вдалося ',
+    'Unfortunately, we are unable to reset your password. Did you entered a valid username? Do you have an email address in your profile?' => 'На жаль, не вдалося скинути пароль. Ви вказали дійсне ім\'я користувача? У вашому профілі задано E-mail?',
```

Comment:
It should be "enter" not "entered". Take a look to this PR https://github.com/kanboard/kanboard/pull/3963

Reply:
oops, thanks. fixed now